src/components/AppBar/components/HamburgerButton/HamburgerButton.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+import { hideVisually } from 'polished';
+import HamburgerIcon from './hamburger.svg';
```

Comment:
There's already a Hamburger component: https://github.com/sumup/circuit-ui/tree/master/src/components/Hamburger. Let's decide on one of them.

Reply:
Hey @connor-baer! I think the idea is to upgrade and use the same component in the future (thinking also on the redesign of the marketing website). 

This new iteration normalizes the horizontal lines to stick more strictly to the hamburger pattern and adds the circle element for increased awareness of the touchable area. Niko design the same hamburger style but for light backgrounds. This could be even a toggle in the future (for adapting the component for both dark and light backgrounds). What do you think?

Reply:
Sounds good to me! I don't care which one we use, just that we settle on one 😉

I like the usability improvements!

Reply:
In that case, could you please extract the Hamburger into a standalone component and replace the old one? We use the hamburger on the website, but separate from the sidebar.

Reply:
Sure can. Do you think we should already replace the other Hamburger before the website redesign?

Reply:
I don't see a reason why not 😊

Reply:
Actually, can this Hamburger turn into a close icon? The existing one does.

Reply:
Hm. I'm somewhat undecided on this. On the one hand upgrading projects is annoying work. At the same time a design system should create consistency and ideally allow consumers to get the latest look for free. I think I lean towards replacing the existing thing, if it's API compatible.

Reply:
Sure! Maybe we could just make some tweaks on the existing one to fix the icon sizes and add the new border. As @joaovtg said, maybe we could add a `type` prop (or something similiar), so we can use on dark and light backgrounds. What do you think?